test/distributed/org/apache/cassandra/distributed/test/PreviewRepairTest.java

```diff
@@ -35,6 +35,7 @@
 
 import com.google.common.util.concurrent.Uninterruptibles;
 
+import org.apache.cassandra.distributed.shared.ClusterUtils;
```

Comment:
nit: import ordering. Have some o.a.c. above and below org.junit here.

Reply:
we really need check style to yell